activestorage/app/controllers/active_storage/blobs/proxy_controller.rb

```diff
@@ -5,8 +5,17 @@ class ActiveStorage::Blobs::ProxyController < ActiveStorage::BaseController
   include ActiveStorage::SetBlob
 
   def show
-    http_cache_forever public: true do
-      send_blob_stream @blob
+    if request.headers["HTTP_RANGE"].present?
+      response.headers["Accept-Ranges"] = "bytes"
```

Comment:
Is there any reason why this is being set here and not in the `send_blob_byte_range_data`?

Reply:
I left it out to keep it in the same place as the one on the other side of the conditional, but it probably does make more sense in that method since if you're sending byte range data you definitely accept byte ranges.

I'll move it